Sources/TuistKit/Generator/GeneratorModelLoader.swift

```diff
@@ -0,0 +1,221 @@
+import Basic
+import Foundation
+import TuistCore
+import TuistGenerator
+
+enum GeneratorModelLoaderError: Error, FatalError {
+    case malformedManifest(String)
+    
+    var type: ErrorType {
+        switch self {
+        case .malformedManifest:
+            return .bug
```

Comment:
Would you say this is a bug? If the model can't be loaded it's an error on the user's end.

Reply:
this is an interesting case - as it stands today, it could very well be a user error :)

Once https://github.com/tuist/tuist/pull/230 lands, such an error would indicate a bug somewhere in our serialization and deserialization of the manifest from Swift. 

You are right though, for now I'll revert this to `.abort` - we can update it later on once the other PR lands.